Sources/MapboxMaps/Viewport/Transitions/Default/DefaultViewportTransition.swift

```diff
@@ -1,7 +1,13 @@
+/// A default ``ViewportTransition`` implementation.
+///
+/// Use ``Viewport/makeDefaultViewportTransition(options:)`` to create instances of this
+/// class.
```

Comment:
should we mention user should keep a reference to the states and transitions for future operations?

Reply:
I'll mention this in the docs for `transition(to:…)` and `defaultTransition`